rust/arrow/src/compute/util.rs

```diff
@@ -71,6 +71,47 @@ pub(super) fn combine_option_bitmap(
     }
 }
 
+/// Compares the null bitmaps of two arrays using a bitwise `or` operation.
+///
+/// This function is useful when implementing operations on higher level arrays.
+pub(super) fn compare_option_bitmap(
```

Comment:
tests missing, I think.

Reply:
Added, I ended up not using `compare_option_bitmap`, but it's a useful function to have